plugins/git/commands/git-cherry-pick-by-patch.md

```diff
@@ -0,0 +1,46 @@
+---
+argument-hint: <commit_hash>
+description: Cherry-pick git commit into current branch by "patch" command
+---
+
+## Name
+git-cherry-pick-by-patch
+
+## Synopsis
+```
+/git-cherry-pick-by-patch commit_hash
```

Comment:
the plugin names precedes the command, then the command name doesn't need the "git" prefix. This would be
`/git:cherry-pick-by-patch`